server/api/index.js

```diff
@@ -191,4 +191,31 @@ router.post("/topics", async (req, res) => {
   }
 });
 
+// // Add User
+router.post("/users", async (req, res) => {
+  const { name, email, password } = req.body;
+  if (
+    (await db.checkUserByNameExist(name)) === false &&
```

Comment:
Here, you are not allowing new users to have the same name. That is not a good idea. The name is not unique, email is.

Reply:
You are right I will change.

Reply:
👍